xml/deployment_scenarios.xml

```diff
@@ -0,0 +1,1175 @@
+        and the <literal>container image registry</literal> for container
+        images.
+       </para>
+       <itemizedlist>
+        <listitem>
+         <para>
+          <literal>1</literal> Host machines for the mirror servers.
+         </para>
+         <itemizedlist>
+          <listitem>
+           <para>
+            SLES 15
+           </para>
+          </listitem>
+          <listitem>
+           <para>
+            2 (v)CPU
+           </para>
+          </listitem>
+          <listitem>
+           <para>
+            8 GB RAM
+           </para>
+          </listitem>
+          <listitem>
+           <para>
+            500 GB Storage
+           </para>
+          </listitem>
+         </itemizedlist>
+        </listitem>
+       </itemizedlist>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term>Internal (Airgapped)</term>
+      <listitem>
+       <para>
+        This machine will host the <literal>&rmtool;</literal> for RPM packages,
+        and <literal>container image registry</literal> for container images
+        as well as a webserver hosting the <literal>Helm chart repository</literal>
+        files.
+       </para>
+       <itemizedlist>
+        <listitem>
+         <para>
+          <literal>1</literal> Host machines for the mirror servers.
+         </para>
+         <itemizedlist>
+          <listitem>
+           <para>
+            SLES 15
+           </para>
+          </listitem>
+          <listitem>
+           <para>
+            2 (v)CPU
+           </para>
+          </listitem>
+          <listitem>
+           <para>
+            8 GB RAM
+           </para>
+          </listitem>
+          <listitem>
+           <para>
+            500 GB Storage
+           </para>
+          </listitem>
+         </itemizedlist>
+        </listitem>
+       </itemizedlist>
+      </listitem>
+     </varlistentry>
+    </variablelist>
+    <important>
+     <title>Adjust Number Of Mirror Servers</title>
+     <para>
+      This scenario description does not contain any fallback contingencies for
+      the mirror servers. Add additional mirror servers (behind a loadbalancer)
+      if you require additional reliability/availability.
+     </para>
+    </important>
+    <procedure>
+     <title>Provision Mirror Servers</title>
+     <step>
+      <para>
+       <link xlink:href="https://www.suse.com/documentation/sles-15/book_quickstarts/data/art_sle_installquick.html">Set
+       up two &sls; 15 machines</link> one on the internal network and one on
+       the airgapped network.
+      </para>
+     </step>
+     <step>
+      <para>
+       Make sure you have
+       <link xlink:href="https://www.suse.com/documentation/sles-15/book_sles_docker/data/preparation.html">enabled
+       the <filename>Containers</filename> module</link> on each server.
+      </para>
+     </step>
+     <step>
+      <para>
+       You will need SSL/TLS certificates to secure services on each server.
```

Comment:
From a customer perspective I would not be sure what I have to do here. Are these certs self-signed or do I have to possibility to obtain them from some already deployed resource?

Reply:
The assumption is air gapped networking is done in an enterprise network. It's the network admins' job to create and provide certificates.

I don't know if there is somebody out there who cares enough about security to want an air gapped network and then use self-signed certs.